docs/stdlib/generic.rst

```diff
@@ -284,17 +284,18 @@ Generic
         db> select contains(b'qwerty', b'42');
         {false}
 
-    When the *haystack* is an :eql:type:`array`, return ``true`` if
-    the array contains the specified element and ``false`` otherwise.
+    When *haystack* is an :eql:type:`array`, the function will return ``true``
+    if the array contains the element specified as *needle* or ``false``
+    otherwise:
 
     .. code-block:: edgeql-repl
 
         db> select contains([2, 5, 7, 2, 100], 2);
         {true}
 
-    When the *haystack* is a :ref:`range <ref_std_range>`, return ``true`` if
-    it contains either the specified sub-range or element and ``false``
-    otherwise.
+    When *haystack* is a :ref:`range <ref_std_range>`, the function will return
+    ``true`` if it contains either the specified sub-range or element, or
```

Comment:
I think I will break this up a little differently to work around it. I find it a little confusing using "and" in a case where two alternatives are mutually exclusive.

Reply:
Does this work?

```rest
    When *haystack* is a :ref:`range <ref_std_range>`, the function will return
    ``true`` if it contains either the specified sub-range or element. The
    function will return ``false`` otherwise.
```